packages/ui-services/src/Import/SimplenoteConverter/SimplenoteConverter.ts

```diff
@@ -15,6 +15,15 @@ type SimplenoteData = {
 const isSimplenoteEntry = (entry: any): boolean =>
   entry.id && entry.content != undefined && entry.creationDate && entry.lastModified
 
+const splitOnce = (str: string, delim: string): string[] => {
+  const i = str.indexOf(delim)
+  if (i < 0) {
+    // no delimeter found, return whole string
```

Comment:
```suggestion
const splitOnce = (str: string, delim: string): string[] => {
  const indexOfDelimiter = str.indexOf(delim)
  const hasDelimiter = indexOfDelimiter > -1
  if (!hasDelimiter) {
```

Given that all of this code is going to be minified anyway, it doesn't make to use shortened variable names like `i` which don't give any context. It should probably be changed to something like `indexOfDelimiter`, along with `str`->`string` and `delim`->`delimiter`.

We always prefer using descriptive variable names instead of using comments. In this case, the conditional can be inverted and extracted into a const with a descriptive name like `hasDelimiter`.